protocol/hardhat.config.ts

```diff
@@ -64,6 +64,15 @@ subtask(TASK_COMPILE_SOLIDITY_GET_SOURCE_PATHS).setAction(
 module.exports = {
   solidity: {
     compilers: [
+      {
```

Comment:
I'm in two minds if we make this a new 0.8.17, or just keep it consistent with the rest of temple core as 0.8.4.

Don't feel strongly tbh 🤷  so this is probably all ok.

Reply:
will leave then